tutorials/using-turing/index.jmd

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This file does not contain any code or examples but is just a landing page on the webpage it seems, so I wonder:
Should this file be added here? Or should it be a regular markdown file? Or at least, do we really have to add Project.toml  + Manifest.toml files here?